src/renderers/dom/fiber/ReactDOMFiberEntry.js

```diff
@@ -531,6 +531,25 @@ function renderSubtreeIntoContainer(
   );
 
   if (__DEV__) {
+    if (container._reactRootContainer && container.nodeType !== COMMENT_NODE) {
+      const hostInstance = DOMRenderer.findHostInstance(
+        container._reactRootContainer.current,
+      );
```

Comment:
Good point - talked more with @spicyj offline also and it looks like we may also just be able to change this spot to filter portals out, and then we might not need a new method?

https://github.com/facebook/react/blob/7b9f64307d7d0d4e9df7f8e2316a2379d7c4bcdf/src/renderers/shared/fiber/ReactFiberTreeReflection.js#L245